contrib/babelfishpg_tds/src/backend/tds/tdsbulkload.c

```diff
@@ -440,10 +448,12 @@ GetBulkLoadRequest(StringInfo message)
 
 		/* Column Name */
 		CheckMessageHasEnoughBytesToRead(&message, sizeof(uint8_t));
+		Assert(sizeof(uint8_t)<=sizeof(colmetadata[currentColumn].colNameLen));
 		memcpy(&colmetadata[currentColumn].colNameLen, &message->data[offset++], sizeof(uint8_t));
 
 		CheckMessageHasEnoughBytesToRead(&message, colmetadata[currentColumn].colNameLen * 2);
 		colmetadata[currentColumn].colName = (char *)palloc0(colmetadata[currentColumn].colNameLen * sizeof(char) * 2 + 1);
+		Assert((colmetadata[currentColumn].colNameLen * 2)<=sizeof(colmetadata[currentColumn].colName));
```

Comment:
This will lead to crash since sizeof(colmetadata[currentColumn].colName)) is 0 (it is still no assigned a value)
so this will always fail and crash.
Why do we need an assert for char*?